src/markdown-pages/build-apps/nr1-deprecations.mdx

```diff
@@ -0,0 +1,197 @@
+
+<Steps>
+
+<Step>
+
+If you already have a local copy of your Nerdpack's repository, make sure you have the latest code.
+
+<>
+
+```bash
+cd <PATH_TO_YOUR_REPOSITORY>
+git pull
+```
+
+<Callout variant="important">
+
+Make sure you replace `<PATH_TO_YOUR_REPOSITORY>` with the real path.
+
+</Callout>
+
+</>
+
+If not, clone your repository.
+
+<>
+
+```bash
+git clone <YOUR_REPO_URL>
+```
+
+<Callout variant="important">
+
+Make sure you replace `<YOUR_REPO_URL>` with your reposority's actual url.
+
+</Callout>
+
+</>
+
+</Step>
+
+<Step>
+
+If you already have the New Relic One CLI, make sure you have the latest version.
+
+<>
+
+```bash
+nr1 update
+```
+
+</>
+
+Otherwise, [install it](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmhvbWUifQ==) so you can test your Nerdpack locally.
+
+</Step>
+
+<Step>
+
+Serve your Nerdpack locally:
+
+<>
+
+```bash
+nr1 nerdpack:serve
+```
+
+</>
+
+</Step>
+
+<Step>
+
+Navigate to your app launcher or custom visualization from the short url at the bottom of your `serve` output:
+
+<>
+
+```bash
+[output] {purple}Launchers:
+[output]  ⁎  {green}launcher {blue}https://onenr.io/0A1bcDEfgHi
+```
+
+</>
+
+</Step>
+
+<Step>
+
+Open your browser's developer console and check for deprecation logs, warnings, and errors:
+
+![Deprecation message](../../images/build-an-app/deprecation-msg.png)
+
+This message tells you not only the code that's affected by the deprecation, but also the due date for updating it.
+
+</Step>
+
+<Step>
+
+Update your code:
+
+<>
+
+```js lineHighlight=3
+return <NrqlQuery
+	pollInterval={NrqlQuery.AUTO_POLL_INTERVAL}
+	accountIds={[12345678]}
+	query="SELECT count(*) FROM Transaction TIMESERIES"
+>
+	{({ data }) => {
+	return <LineChart data={data} />;
+	}}
+</NrqlQuery>;
+```
+
+</>
+
+In this example, the `accountId` property for `NrqlQuery` is changing to `accountIds`. For your actual Nerdpack, the deprecation message will tell you what you need to change.
+
+While still serving your application, your Nerdpack updates automatically in the browser when you save your file.
+
+</Step>
+
+<Step>
+
+Check your console again:
+
+![No deprecation message](../../images/build-an-app/no-deprecation-msg.png)
+
+When you fix your code, your console no longer logs a deprecation message.
+
+</Step>
+
+<Step>
+
+Commit your code and push it to your remote repository:
+
+<>
+
+```bash
+git commit -am "fix: deprecated code"
+git push origin
+```
+
+</>
+
+</Step>
+
+<Step>
+
+[Publish your updated Nerdpack](/build-apps/publish-deploy/publish/) to the New Relic One catalog.
+
+</Step>
+
+</Steps>
+
+We strive to write clear messages so you know when and how you need to change your code to address deprecated features.
+
+However, it may still be valuable for you to reach out to our community channels if you need further assistance.
```

Comment:
Maybe we can make a call out that ask users to let us know if the error messages aren't clear by posting in the github discussions linked below.